packages/elements/src/internals/machines/sign-in/start.machine.ts

```diff
@@ -38,7 +38,10 @@ export const SignInStartMachine = setup({
     ),
   },
   actions: {
-    sendToNext: ({ context }) => context.parent.send({ type: 'NEXT' }),
+    sendToNext: ({ context, event }) => {
+      // @ts-expect-error -- We're calling this in onDone, and event.output exists on the actor done event
+      return context.parent.send({ type: 'NEXT', resource: event?.output });
```

Comment:
If anyone wants to take a crack at fixing this type that would be appreciated 😅 My xstate-ts-fu isn't up to par.

Reply:
This works for now. I’ll likely update to params.